docs/guide/deployer.md

```diff
@@ -150,18 +150,18 @@ For opting in to App, `deployer` supports the following methods:
    Check our [examples/htlc-pyteal-ts](https://github.com/scale-it/algo-builder/tree/master/examples/htlc-pyteal-ts) project to explore how to deploy Stateless Smart Contracts(lsig). In the file `scripts/deploy.ts`, you will find:
 
```

Comment:
let's add a section above this one to explain about to ways of managing artifacts: 
* using names
* using code